data_architect_misc/data_transformer/transform_functions/common_comp_harm_transform_functions.py

```diff
@@ -826,6 +855,16 @@ def filter_and_rearrange_columns_for_final_output(self,
             df,
             comp_harm_constants.EXPECTED_COLUMNS
         )
+    def multiply_values_in_given_column_by_thousand(
```

Comment:
I changed my mind. Let's rename this function to a simpler one: "multiply_values_in_column_by_a_thousand(...)"

Reply:
Renamed.

Reply:
Done